extensions/LIZARD_OFFICIAL/2FA.js

```diff
@@ -0,0 +1,37 @@
+
+  "use strict";
+
+  class MFA {
+    getInfo() {
+      return {
+        id: "twofactorauth",
+        name: "2FA",
+        blocks: [
+          {
+            opcode: "gettotpcode",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "GET TOTP CODE [CODE]",
```

Comment:
This is more of a nitpick, but to parody blocks in Scratch it's preferred to use lowercase for your blocks unless necessary.